src/test/kotlin/com/terraformation/backend/species/SpeciesServiceTest.kt

```diff
@@ -80,6 +84,26 @@ internal class SpeciesServiceTest : DatabaseTest(), RunsAsUser {
           match { it.scientificName == "New name" },
       )
     }
+
+    eventPublisher.assertEventPublished(
+        SpeciesEditedEvent(
+            species =
+                SpeciesModel(
+                    averageWoodDensity = null,
+                    checkedTime = null,
+                    commonName = null,
+                    conservationCategory = null,
+                    dbhSource = null,
+                    dbhValue = null,
+                    deletedTime = null,
+                    ecologicalRoleKnown = null,
+                    ecosystemTypes = emptySet(),
+                    familyName = null,
+                    growthForms = emptySet(),
```

Comment:
Could remove these since they're the default values in the `SpeciesModel` constructor and this call is already using other default values.